src/components/browse/components/FacetList/FacetTermsList.js

```diff
@@ -117,18 +117,20 @@ export class Term extends React.PureComponent {
     }
 
     render() {
-        const { term, facet, status, termTransformFxn, isFiltering } = this.props;
+        const { term, facet, status, termTransformFxn, isFiltering, termIconStyle } = this.props;
+        
         const selected = (status !== 'none');
         const count = (term && term.doc_count) || 0;
+        const isRadio = termIconStyle === 'radio'; 
```

Comment:
Try to avoid passing down string props if can, in favor of boolean, if what we ultimately need is boolean (better performance in memoized components). Could probably simplify this and just have either `props.useRadioIcon` (w defaultProps.useRadioIcon=false).

Reply:
@alexkb0009  Yes, definitely makes sense. Replaced `termIconStyle` w/ `useRadioButton`.